tests/api-watch-diff/dune

```diff
@@ -0,0 +1,6 @@
+(library
+ (name ppx_expect_test)
```

Comment:
Since we only have one module here, it's best to have this match the name of the test file.

Reply:
Just want to confirm this:
```
(library
 (name test)
```
This should be the new name right? (After I rename test file from ```ppx_expect.ml``` to ```test.ml```)

Reply:
yes exactly!